Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -2432,10 +2448,17 @@ WebContextMenuProxyGtk* webkitWebViewBaseGetActiveContextMenuProxy(WebKitWebView
     return webkitWebViewBase->priv->activeContextMenuProxy;
 }
 
-GdkEvent* webkitWebViewBaseTakeContextMenuEvent(WebKitWebViewBase* webkitWebViewBase)
+#if USE(GTK4)
+GRefPtr<GdkEvent> webkitWebViewBaseTakeContextMenuEvent(WebKitWebViewBase* webkitWebViewBase)
 {
-    return webkitWebViewBase->priv->contextMenuEvent.release();
+    return std::exchange(webkitWebViewBase->priv->contextMenuEvent, nullptr);
 }
+#else
+GUniquePtr<GdkEvent> webkitWebViewBaseTakeContextMenuEvent(WebKitWebViewBase* webkitWebViewBase)
+{
+    return WTFMove(webkitWebViewBase->priv->contextMenuEvent);
```

Comment:
Um, I don't actually know whether this works because I only tested a GTK 4 build... pretty sure EWS will notice if there is a problem, though.

Reply:
The bot is happy.

Reply:
Ah, GCC is complaining about this with a local build, so I guess it's not needed even for move-only types. Not sure why EWS didn't care.